src/v/security/audit/audit_log_manager.h

```diff
@@ -193,6 +196,7 @@ class audit_log_manager
     /// Other references
     cluster::controller* _controller;
     kafka::client::configuration& _config;
+    std::unique_ptr<audit_probe> _probe;
```

Comment:
Reason this is a `std::unique_ptr<probe>` instead of `audit_probe` ?

Reply:
not really, no

Reply:
> not really, no

Check git history, probes we're wholesale moved to `unique_ptr` due to capturing `this` and then moves.